pkg/apis/autoscaling/register.go

```diff
@@ -83,6 +83,19 @@ const (
 	// scale-to-zero-pod-retention-period global setting.
 	ScaleToZeroPodRetentionPeriodKey = GroupName + "/scaleToZeroPodRetentionPeriod"
 
+	// MetricAggregationAlgorithm is the annotation that can be used for selection
+	// of the algorithm to use for averaging metric data in the Autoscaler.
```

Comment:
Are we going to apply the same algorithm to both windows or just one? If both, will it always be the same algorithm for both or would there be a use case for a diff algorithm per window? (Obviously asking because we may want to name this StableWindowAggregationAlgorithm, for example, if so).

Reply:
Interestingly, it is probably less important for the panic window, since they are shorter and difference between regular and weighted avg is small.